packages/connect-common/files/coins.json

```diff
@@ -2279,7 +2299,9 @@
         {
             "blockchain_link": {
                 "type": "blockfrost",
-                "url": ["wss://trezor-cardano-preview.blockfrost.io"]
+                "url": [
+                    "wss://wiser-green-owl.solana-mainnet.discover.quiknode.pro/c218d0c9e451f6c5c0bf8b7c7d8e5a72384c9570/"
```

Comment:
what is this?

Reply:
Ups, sorry, obviously a mistake. Thanks for noticing, I will fix that.

Reply:
[fixup! feat(common): add tmp solana coin to coins.json](https://github.com/trezor/trezor-suite/pull/9256/commits/b148f3994db26d89534625dd4fa807a2d438345a)